game/messages.py

```diff
@@ -2089,3 +2091,53 @@ def description_level109():
 def hint_level109():
     message = ugettext("To halve the value of a variable you can do something like the following:<br><pre>n /= 2</pre>")
     return message
+
+
+def _title_episode1():
+    return ugettext('Getting Started')
```

Comment:
can't this be populated from the database or the API? https://www.codeforlife.education/rapidrouter/api/episodes/1/